protocol/v2/ssv/runner/proposer.go

```diff
@@ -155,6 +172,10 @@ func (r *ProposerRunner) ProcessPostConsensus(signedMsg *specssv.SignedPartialSi
 		return nil
 	}
 
+	metricsPostConsensusDuration.
+		WithLabelValues(hex.EncodeToString(r.GetShare().ValidatorPubKey), spectypes.BNRoleProposer.String()).
```

Comment:
we are doing too many hex encoding, you can do it once and reuse

Reply:
@amir-blox I agree that we do it many times, but I'm not sure how to have a good design for reusing, could you please suggest one?

Reply:
Define it once, and use the variables instead of invoking the same functions again:
```go
pkHex, beaconRole := hex.EncodeToString(r.GetShare().ValidatorPubKey), spectypes.BNRoleProposer.String()

metric.WithLabelValue(phHex, beaconRole)...
```

Reply:
thanks, done